src/main/java/core/basesyntax/service/UserPasswordValidationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class UserPasswordValidationException extends UserValidationException {
+    public UserPasswordValidationException(String message) {
+        super(message);
+    }
```

Comment:
it might be better to create one class and name it as RegistrationException ?

Reply:
> it might be better to create one class and name it as RegistrationException ?

I want to make it clear. If my test fails from password user must know it. There is no other way except reading message to indicate it. It might be useful. I don't know is it widespread but I'll try to commit like it is now.